.tx/integration_config.yml

```diff
@@ -2,12 +2,12 @@ filters:
 
   - filter_type: file
     source_language: en
-    source_file: Client/Ecosia/L10N/en.lproj/Ecosia.strings
+    source_file: Ecosia/L10N/en.lproj/Ecosia.strings
     file_format: STRINGS
     translation_files_expression: 'Client/Ecosia/L10N/<lang>.lproj/Ecosia.strings'
 
   - filter_type: file
     source_language: en
-    source_file: Client/Ecosia/L10N/en.lproj/Plurals.stringsdict
+    source_file: Ecosia/L10N/en.lproj/Plurals.stringsdict
     file_format: STRINGSDICT
-    translation_files_expression: 'Client/Ecosia/L10N/<lang>.lproj/Plurals.stringsdict'
+    translation_files_expression: 'Client/Ecosia/L10N/<lang>.lproj/Plurals.stringsdict'
```

Comment:
Shouldn't this be without `Client` folder as well?

Reply:
Awesome catch! 🚀